Dockerfile

```diff
@@ -3,6 +3,9 @@ FROM debian:wheezy
 # add our user and group first to make sure their IDs get assigned consistently, regardless of whatever dependencies get added
 RUN groupadd -r rabbitmq && useradd -r -d /var/lib/rabbitmq -m -g rabbitmq rabbitmq
 
+RUN apt-key adv --keyserver keyserver.ubuntu.com --recv D208507CA14F4FCA
```

Comment:
Can we get the full fingerprint here and use `pool.sks-keyservers.net` for better reliability?